### DISCUSSION TITLE
When changing a list's name, you are sent to the old slug

### DISCUSSION THREAD

**Discussion by MattIPv4** [community/community#8233] [Other Feature Feedback, Questions, & Ideas] [answered]
1. Create a list
2. Go to the list page
3. Edit the list with a new name
4. Save, observe sent to old list URL

**Comment by brianlovin** [comment, marked as answer]
This should be fixed! Thanks @MattIPv4

**Reply by MattIPv4** [reply]
✅ Thank you!
